gslib/util.py

```diff
@@ -704,6 +704,12 @@ def GetCertsFile():
           configured_certs_files.append(fname)
           disk_certs_file = fname
       certs_file = disk_certs_file
+  # The 'system' keyword indicates to use the system installed certs. Some
+  # Linux distributions patch the stack such that the Python SSL
+  # infrastructure picks up the system installed certs by default, thus no
+  #  action necessary on our part
+  if certs_file == 'system':
```

Comment:
Since this is mutually exclusive with the code block above, could you please move it to the top of the function and just return "None" there?